java-manta-client/src/main/java/com/joyent/manta/client/multipart/AbstractMultipartManager.java

```diff
@@ -197,7 +198,8 @@ public PART uploadPart(final UPLOAD upload,
      */
     abstract PART uploadPart(UPLOAD upload,
                              int partNumber,
-                             HttpEntity entity) throws IOException;
+                             HttpEntity entity,
+                             HttpContext context) throws IOException;
```

Comment:
Missing javadoc on `context`?  I had some nullability questions below, but they might be better addressed with prose here.